src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
+        Random randomBall = new Random();
+        int value = randomBall.nextInt(100);
```

Comment:
Magic number 100: It would be better to declare a constant for this value to improve code readability and maintainability.